src/util.js

```diff
@@ -67,21 +67,34 @@ export const distanceBetweenCoordinates = (c1, c2) => {
 /**
  * Format a distance in meters into a human-readable string with unit.
  *
- * This only supports m / km for now, but could read a config option and return
- * ft / mi.
- *
  * @param {Number} distance Distance in meters
  * @returns {String} Formatted string including unit
  */
 export const humanReadableDistance = (distance) => {
-  let unit = "m";
-  if (Math.abs(distance) >= 1000) {
-    distance = distance / 1000;
-    unit = "km";
+  let unit = 'm';
+  let digits = 1;
+
+  if(config.units == "imperial") {
```

Comment:
Please add a test for this to https://github.com/owntracks/frontend/blob/7877aaa9f833aa12ff78c7f8690658a8ee5a1a0c/tests/util.test.js#L116-L133